mappings/net/minecraft/enchantment/EnchantmentsEntry.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_1889 net/minecraft/enchantment/EnchantmentsEntry
```

Comment:
Perhaps use the singular `EnchantmentEntry` instead?

Reply:
enchantments refers to the nbt list Enchantments or StoredEnchantments. This object is part of that list rather than part of the current Enchantment we have, so I named it with an s.

Reply:
Done, thanks earth!